src/modules/nav/nav.component.ts

```diff
@@ -22,7 +22,7 @@ export class StacheNavComponent implements OnInit, StacheNav {
 
   public constructor(
     private router: Router,
-    private windowRef: SkyAppWindowRef) {}
+    private windowRef: SkyAppWindowRef) { }
```

Comment:
Some linters require spaces between curly brackets. This is a convention I've seen on many Angular 2 docs.